guides/it/software_licenses.md

```diff
@@ -18,6 +18,8 @@ If you would like to request new software that Made Tech doesn't currently use a
 ## Standard Software
 If you need access to existing software email [operations@madetech.com](mailto:operations@madetech.com) or drop a Slack in #Ops (except where indicated). This is not an exhaustive list (we're still working on that).
 
+It's worth remembering that you're likely to need admin access to install or most apps. Team members can request access to admin permissions by emailing [support@systemagic.co.uk](mailto:support@systemagic.co.uk) requesting admin access with details of the purpose for access e.g upgrading system software. Access should be used only in line with the purpose detailed in the request. You must not log into the admin account on your laptop directly.
```

Comment:
```suggestion
It's worth remembering that you're likely to need admin access to install most software. Team members can request access to admin permissions by emailing [support@systemagic.co.uk](mailto:support@systemagic.co.uk) requesting admin access with details of the purpose for access e.g upgrading system software. Access should be used only in line with the purpose detailed in the request. You must not log into the admin account on your laptop directly.
```

Reply:
Might be worth mentioning that it's 09:00 to 17:00 to manage expectations around getting that admin password.